src/mokka/src/main/java/pl/hycom/mokka/endpoint/FileController.java

```diff
@@ -54,11 +54,16 @@
     @ResponseBody
     public ResponseEntity<FileSystemResource> fetchFile(
             @PathVariable("file-id")
-                    String fileId) throws FileNotFoundException {
-        LOG.debug("Calling FileController#fetchFile with arguments [" + fileId + "]");
+                    String fileId,
+            @RequestParam(name = "cd",
```

Comment:
"cd" does not say much about parameter's purpose. I would go with full name (without abbreviation)